using System;
using System.Runtime.ExceptionServices;
/*
namespace codeflow
{
    internal class Exer1
    {

        static void Main(string[] args)
        {
            for(int i = 1; i <= 5; i++)
            {
                for (int j = 1; j <= i; j++)
                {
                    Console.Write('*');
                }
                Console.WriteLine();
            }
        }
    }
}*/
